test/remote/base_path_tests.js

```diff
@@ -4,12 +4,12 @@
 
 var test = require('../ptaptest')
 var TestServer = require('../test_server')
-var Client = require('../client')
+var Client = require('../client/index')
```

Comment:
Not a comment to stop this PR but I just want to note that I disapprove of purely-cosmetic changes like this as a general principle. There are enough opportunities for merge conflicts as it is, without us adding new ones that make no difference to how stuff runs. If something improves code clarity that's fair enough but I don't see this in that category.